### ISSUE TITLE
Web Client not actively showing changes in display widgets, without browser refresh.

### ISSUE THREAD

**Issue by Gun-neR** [avin/blynk-web-client#4]
NO device pushed action is automatically registered in this Web Client, I must refresh web page to see changes in display widgets from `virtualWrite()` `setProperty()`, etc.

**Comment by avin** [comment]
Yup, there is a problem, because I'm using device-API (and the server recognizes me as a device). I will try to dig in into blynk protocol to resolve this problem. Currently this is the biggest problem of this client :(

**Comment by Gun-neR** [comment]
No problem... I am not being nitpicky, I had just noticed that no one else seems to be posting anything, so I figured I can beta test for you :P

**Comment by avin** [comment]
I did not find any official API method to subscribe for pin values changes, so I made temporary solution with `Sync` button (it uses SYNC_HW under the hood) and AutoSync timer to do sync operations automatically.

**Comment by Gun-neR** [comment]
1 second+ refresh is perfect for my typical uses.  Looking great!

One minor glitch I noticed after the first second of operation (regardless of set sync time), the UTF-8 image gets garbled.  Sync related?

![screenshot 2019-02-07 12 31 17](https://user-images.githubusercontent.com/28169425/52441042-84be2d00-2ad4-11e9-841d-43047b545e47.png)
![screenshot 2019-02-07 12 31 17 2](https://user-images.githubusercontent.com/28169425/52441043-8556c380-2ad4-11e9-90ac-e1bef778e84f.png)

**Comment by avin** [comment]
I'll check it. First time I get values from a project object, and when syncing - it comes from websockets in a binnary format, may be while I convert it - something goes wrong

**Comment by avin** [comment]
Seems like it is blynk-server problem. First time i created this widget - it was OK, but today I see the problem and wrong data comes from server :\

![image](https://user-images.githubusercontent.com/36100/53785277-fdc07100-3f28-11e9-9479-0ce019f216b4.png)

**Comment by Gun-neR** [comment]
Hmm, originally I was using code to generate the symbol, which worked for the first display, just not after...  It is on a 1 second timer.

`Blynk.virtualWrite(V7, String(WiFi.RSSI()) + String(" \xF0\x9F\x93\xB6"));`

Then I changed it to the same emoticon, but picked from the phone and placed in the Labeled Display Widget, and that works fine every second.

`Blynk.virtualWrite(V7, String(WiFi.RSSI()));`

![image](https://user-images.githubusercontent.com/28169425/53786876-dd18fc80-3ed1-11e9-8d2c-f335c82d232c.png)

![image](https://user-images.githubusercontent.com/28169425/53786931-09347d80-3ed2-11e9-8689-e3d45773efcc.png)

Go figure :D

**Comment by avin** [comment]
Seems like it is a floating problem :\
